### ISSUE TITLE
I found a way to arrange the dates!!

### ISSUE THREAD

**Issue by LANZATURCA** [zincplusplus/habit-tracker#38]
Sorry for the bad translation, this is my first contribution about this

As you can see, July has 31 days, you just have to look at the calendar and check how many days the month brings

![Captura de pantalla 2024-07-18 224007](https://github.com/user-attachments/assets/2101b1e4-b251-4b87-87f4-39ecb65ff51a)

1.  Add this code

```habittracker
{
	"path": "name of your folder/",
		 "lastDisplayedDate": "2024-07-31",
		"daysToShow": 31 	 
}
```

2.  Change the word "Name of your folder" to the directory where you have empty habit notes

I'll show you an example of how I'll do everything step by step with the month of November, If you notice, November of this year has 30 days, we only have to modify certain parameters in the code

![Captura de pantalla 2024-07-18 225211](https://github.com/user-attachments/assets/3b56f46b-6610-48ee-afaf-517c9b4d8e82)

1.

```habittracker
{
	"path": "name of your folder",
		 "lastDisplayedDate": "2024-11-30",
		"daysToShow": 30 	 
}
```

2. Just change the name folder

![Captura de pantalla 2024-07-18 225224](https://github.com/user-attachments/assets/e62f5455-b70d-4bf7-bbe4-6e49179778e2)

**Comment by zincplusplus** [comment]
@LANZATURCA I'm sorry but it's hard to follow what bug you're talking about. I can understand Spanish, so if you can, could you please send your message again in your native tongue? I'd really wanna hear from you 🙏

**Comment by kulbhushanchand** [comment]
@zincplusplus I understand what @LANZATURCA was trying to convey. It is a very good way to organize the habit tracker month-wise.

In a nutshell, to display a particular month only, one can set the `lastDisplayedDate` to the last date of the month and `daysToShow` to the number of days on that month. Therefore the tracker will show the data of that month only.

 Also, in this way one can embed different trackers for different months.

**Comment by zincplusplus** [comment]
This makes sense. I'm leaving a note here to add 2 new properties: startDate and endDate ( make sure this works also if lastDisplayedDate is specified, the legacy name )
